### PR TITLE
Turkish Translation

### PR DESCRIPTION
**Pull request by r3dmc**

### CODE REVIEW

**Comment by r3dmc** [comment]
Sorry for last PR I forgot commiting resources files. I didn't translate comments in configs etc. because I want to use this translation in my server ASAP but, I'will commit them later.

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 1]
```diff
@@ -463,6 +502,7 @@
             put(Language.RU, "Не найдены связанные каналы");
             put(Language.ET, "Ühtegi kanalit pole ühendatud");
             put(Language.ZH, "未連接任何頻道");
+            put(Language.EN, "Hiçbir kanal bağlanmadı");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: Wrong language

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 2]
```diff
@@ -320,6 +346,7 @@
             put(Language.RU, "DiscordSRV API Listener {listenername} сгенерировал ошибку");
             put(Language.ET, "DiscordSRV API Listener {listenername} tagastas veateate");
             put(Language.ZH, "DiscordSRV API Listener {listenername} 發生錯誤");
+            put(Language.EN, "DiscordSRV API Dinleyicisi {listenername} bir hata çıkardı");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: Wrong language

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 3]
```diff
@@ -287,6 +310,7 @@
             put(Language.RU, "Отключение завершено за {ms}мс");
             put(Language.ET, "Väljalülitus teostatud {ms}ms jooksul");
             put(Language.ZH, "伺服器已關閉，耗時{ms}ms");
+			put(Language.TR, "Kapatım {ms}ms içerisinde gerçekleşti");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: No tabs please

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 4]
```diff
@@ -298,6 +322,7 @@
             put(Language.RU, "API listener {listenername} подписан на ({methodcount} методы)");
             put(Language.ET, "API listener {listenername} on kuulamas ({methodcount} meetodit)");
             put(Language.ZH, "API listener {listenername} 已訂閱 ({methodcount} 種方案)");
+			put(Language.TR, "API dinleyicisi {listenername} ({methodcount} kadar methoda abone oldu)");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: No tabs please

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 5]
```diff
@@ -276,6 +298,7 @@
             put(Language.RU, "Неверный ID канала для перенаправления вывода консоли, сообщения консоли не будут пересылаться");
             put(Language.ET, "Konsoolikanali ID oli sobimatu, konsooli väljundit ei edastata");
             put(Language.ZH, "錯誤的控制台頻道ID, 並未轉送控制台輸出。");
+			put(Language.TR, "Konsol kanal ID'si geçersiz, konsol çıktısı yönlendirilmiyor");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: No tabs please

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 6]
```diff
@@ -265,6 +286,7 @@
             put(Language.RU, "Сервер найден");
             put(Language.ET, "Server leitud");
             put(Language.ZH, "已找到伺服器");
+            put(Language.EN, "Sunucu bulundu");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: Wrong language

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 7]
```diff
@@ -495,6 +537,7 @@
             put(Language.ES, "Vault no está instalado. Se necesita para que la sincronización de grupo funcione con zPermissions. Instala Vault si quieres esta característica.");
             put(Language.RU, "Vault не установлен. Он требуется для синхронизации групп zPermissions. Установите Vault, если вы хотите пользоваться этой функцией.");
             put(Language.ET, "Vault ei ole paigaldatud. See on nõutud gruppide sünkroonimise toimimiseks zPermissions-iga. Paigalda Vault, kui soovid seda funktsiooni.");
+            put(Language.TR, "Vault kurulmamış. Z-permission ile grup senkronizasyonunun çalışması için bu gerekli. Bu özelliği istiyorsız Vault'u kurun.");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: Move this one line down

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 8]
```diff
@@ -518,6 +562,7 @@
             put(Language.RU, "Ошибка логирования действий консоли в");
             put(Language.ET, "Esines viga konsoolitegevuse logimisel asukohta");
             put(Language.ZH, "控制台記錄錯誤");
+            put(Language.EN, "Konsol eylemini kaydederken hata oluştu");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: Wrong language

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 9]
```diff
@@ -584,6 +634,7 @@
             put(Language.RU, "Сохранение метрик было отменено, ввиду отсутствия оных");
             put(Language.ET, "Näitajate salvestamine jäeti vahele, kuna näitajad puudusid");
             put(Language.ZH, "由於沒有任何Metrics， 已略過儲存。");
+            put(Language.EN, "Hiç olmadığı için metriklerin kaydı atlandı");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: Wrong language

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 10]
```diff
@@ -207,6 +223,7 @@
             put(Language.RU, "Вывод консоли успешно перенаправлен в канал");
             put(Language.ET, "Konsooliedastus on kanalile määratud");
             put(Language.ZH, "控制台轉送已指派至頻道");
+			put(Language.TR, "Konsol, atanan kanala yönlendiriyor");
```
Comment (Vankka) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: Tab

**Comment by Scarsz** [comment]
Closing due to being outdated, feel free to update & resubmit

**Pull request closed without merging**